samples/cpp/benchmark_app/main.cpp

```diff
@@ -863,9 +863,9 @@ int main(int argc, char* argv[]) {
         for (auto&& ds : device_nstreams) {
             try {
                 const std::string key = getDeviceTypeFromName(ds.first) + "_THROUGHPUT_STREAMS";
-                device_nstreams[ds.first] = core.get_property(ds.first, key).as<std::string>();
+                device_nstreams[ds.first] = compiledModel.get_property(key).as<std::string>();
             } catch (const ov::Exception&) {
-                device_nstreams[ds.first] = core.get_property(ds.first, ov::num_streams.name()).as<std::string>();
+                device_nstreams[ds.first] = compiledModel.get_property(ov::num_streams.name()).as<std::string>();
             }
```

Comment:
@ilya-lavrenov  why do we still have to keep code to query both 1.0 and 2.0 key in sample application? especially this THROUGPUT_STEAMS key has already been mapped to ov::num_streams in cpu/gpu plugin

Reply:
I don't know.
I am not responsible for benchmark_app and how it's written 

CC @nkogteva @Wovchena

Reply:
It is done for historical reason: moving to new API in BA was done before all plugins support new properties API.